File: citpwawcl/src/app/models/request-terzo-responsabile.ts
import { DatiImpresa } from '@sigit/common-lib';
import { Autodichiarazione } from './dati-autodichiarazione';
import { DatiContratto } from './dati-contratto';
import { DatiImpianto } from './dati-impianto';
import { UtenteLoggato } from './utente-loggato';

export class RequestTerzoResponsabile{
    datiImpianto: DatiImpianto;
    utenteLoggato: UtenteLoggato;
    datiContratto: DatiContratto;
    autodichiarazioni: Autodichiarazione[];
    datiImpresa?: DatiImpresa;
}

File: sigitext/src/java/it/csi/sigit/sigitext/dto/sigitext/RequestTerzoResponsabile.java
package it.csi.sigit.sigitext.dto.sigitext;

import java.io.Serializable;
import java.util.List;

public class RequestTerzoResponsabile implements Serializable {
	
	/**
	 * 
	 */
	private static final long serialVersionUID = 1L;
	
	private DatiImpianto datiImpianto;
	private UtenteLoggato utenteLoggato;
	private DatiContratto datiContratto;
	private List<Autodichiarazione> autodichiarazioni;
	private DatiImpresa datiImpresa;
	
	
	public DatiImpianto getDatiImpianto() {
		return datiImpianto;
	}
	public void setDatiImpianto(DatiImpianto datiImpianto) {
		this.datiImpianto = datiImpianto;
	}
	public UtenteLoggato getUtenteLoggato() {
		return utenteLoggato;
	}
	public void setUtenteLoggato(UtenteLoggato utenteLoggato) {
		this.utenteLoggato = utenteLoggato;
	}
	public DatiContratto getDatiContratto() {
		return datiContratto;
	}
	public void setDatiContratto(DatiContratto datiContratto) {
		this.datiContratto = datiContratto;
	}
	public List<Autodichiarazione> getAutodichiarazioni() {
		return autodichiarazioni;
	}
	public void setAutodichiarazioni(List<Autodichiarazione> autodichiarazioni) {
		this.autodichiarazioni = autodichiarazioni;
	}
	public DatiImpresa getDatiImpresa() {
		return datiImpresa;
	}
	public void setDatiImpresa(DatiImpresa datiImpresa) {
		this.datiImpresa = datiImpresa;
	}
	
	
}
